File: class_tool/src/main/java/com/tempsfteam/class_tool/vo/ExperimentInfoVO.java
package com.tempsfteam.class_tool.vo;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * @author : IMG
 * @create : 2024/10/17
 */
@Data
@NoArgsConstructor
@AllArgsConstructor
public class ExperimentInfoVO {

    private Integer experimentId;

    private String name;

    private Integer courseId;

    private String courseName;

    private String courseDescription;

    private String courseIcon;
}

File: class_tool/src/main/resources/static/src/models/vo/experimentInfoVO.js
export default {
    name: 'experimentInfoVO',
    title: 'ExperimentInfoVO',
    idField: 'experimentId',
    fields: [
        { name: 'experimentId', label: '实验ID', type: 'number' },
        { name: 'name', label: '实验名', type: 'string' },
        { name: 'courseId', label: '课程ID', type: 'number' },
        { name: 'courseName', label: '课程名', type: 'string' },
        { name: 'courseDescription', label: '课程描述', type: 'string' },
        { name: 'courseIcon', label: '课程图标', type: 'string' }
    ]
};